src/data.ts

```diff
@@ -134,25 +134,30 @@ async function getLinkedNotes(source_id:string, maxDegree:number) : Promise<Map<
     visited.push(...pending);
     pending = [];
 
-    notes.forEach(note => {
+    for (const note of notes) {
```

Comment:
Is the only change here making note `const` or do these these two ways of looping over each have different behavior in JS?

Reply:
No, the change from `forEach` to `for...of` is needed to make `await` in https://github.com/treymo/joplin-link-graph/pull/50/commits/2cef281d075f9370c504bc50a8018eae6792d7ed#diff-00236e8c5b72b2947db56a33c5402dd2c3e60f8b1ac5b1178b07046ff012accfR180 blocking. `forEach` would not wait for the backlinks result.